StPetersburg/InEtn210325.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0"
+     xml:id="InEtn210325"
+     xml:lang="en"
+     type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>First Epistle of John</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens: Eine multimediale
+                            Forschungsumgebung / beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+              <msDesc type="scroll">
+                <msIdentifier>
+                    <repository ref="INS0524InEtn"/>
+                    <idno>2103-25</idno>
+                    <altIdentifier>
+                      <idno>Platonov cat. III.116</idno>
+                    </altIdentifier>
+                </msIdentifier>
+                <msContents>
+                  <summary>
+                    The manuscript was used to train reading or copying.
```

Comment:
Interesting information! I assume that the information is given like this in the catalogue, without explaining why the cataloguer came to this conclusion? I am wondering whether this is the best place for it, but have no other idea either. What do you think @PietroLiuzzo @DenisNosnitsin1970

Reply:
I would say, "according to "ref to the note", the manuscript was meant for disciples of a nebab-school. I think the passage is exactly the one with what they start, to learn how to read letters (it can /should be checked).

Reply:
In this case, there is a note which says that the manuscript was produced for learning, I can add a ref. But in the case of InEtn210321 this information is just mentioned in the catalogue description.

Reply:
I am sorry, but I really don't think that `<summary>` is the appropriate place for this. It should be used for a summary of the content, if at all. At least in this case, the information could be added in the `<desc>` of a1. As for the manuscript where it is just mentioned by the catalogue, maybe an extra? Or a note in the msItem?